plugins/modules/mso_schema_template_external_epg.py

```diff
@@ -114,6 +114,12 @@
     description:
     - Preferred Group is enabled for this External EPG or not.
     type: bool
+  qos_level:
```

Comment:
mso_schema_template_contract_filter module also has qos_level, should we put the (choices: [ unspecified, level1, level2, level3, level4, level5, level6 ]) in constants.py?